pattern-2/bosh-release/jobs/gateway/templates/repository/conf/datasources/master-datasources.xml

```diff
@@ -92,26 +94,27 @@
             </definition>
         </datasource>
 
-         <datasource>
-            <name>WSO2AM_STATS_DB</name>
-            <description>The datasource used for getting statistics to API Manager</description>
-            <jndiConfig>
-                <name>jdbc/WSO2AM_STATS_DB</name>
-            </jndiConfig>
-            <definition type="RDBMS">
-                <configuration>
-                    <url><%= p("wso2apim.am_db.jdbc_url") %></url>
```

Comment:
Why are we removing parameters from stats DB?

Reply:
Because we have disabled stats as it is not used by almost all the users.